### PR TITLE
WIP: Use ltp_known_issues.json in publiccloud LTP test runs

### PR DESCRIPTION
**Pull request by grisu48**
Known issues in LTP test runs are stored as json in the configurable LTP_KNOWN_ISSUES setting.
We now use this json to mask known failures as softfailures.

- Related ticket: https://progress.opensuse.org/issues/92470
- Verification run: http://duck-norris.qam.suse.de/t6598

### CODE REVIEW

**Comment by grisu48** [comment]
This PR is ready to be discussed, but we need to wait for the VR before merging. I'm still expecting some minor changes due to VR failure.

**Review by asmorodskyi — CHANGES_REQUESTED** [review]
Let's discuss it before merging

**Comment by grisu48** [comment]
We agreed to leave this PR open as an alternative and @asmorodskyi will try until tomorrow to find a native perl solution instead. If we don't find one, then we might proceed with this one.

Please do not merge until further notice.

**Pull request closed without merging**
